src/lib/Routes.ts

```diff
@@ -0,0 +1,66 @@
+export enum Routes {
+    Dashboard = "/dashboard",
+    Notification = "/notification",
+    Attendance = "/attendance",
+    MarkAttendance = "/mark-attendance",
+    Fund = "/fund",
+    Member = "/member",
+    AddMember = "/add-member",
```

Comment:
This is wrong route. It should be `/member/add` and would be placed in members folders nestedly.

Reply:
Fixed